project-base/storefront/components/Blocks/Product/Filter/DeferredFilterPanel.tsx

```diff
@@ -0,0 +1,15 @@
+import { FilterPanelProps } from './FilterPanel';
+import { SkeletonModuleFilterPanel } from 'components/Blocks/Skeleton/SkeletonModuleFilterPanel';
+import dynamic from 'next/dynamic';
+import { useDeferredRender } from 'utils/useDeferredRender';
+
+const FilterPanel = dynamic(() => import('./FilterPanel').then((component) => component.FilterPanel), {
+    ssr: false,
+    loading: () => <SkeletonModuleFilterPanel />,
+});
+
+export const DeferredFilterPanel: FC<FilterPanelProps> = (props) => {
```

Comment:
DeferredXXX - i am thinking about switch naming - and let word 'deferred' to be last. Same as skeleton - i thought we are using this keyword at the end. For better orientation and search.

Reply:
and for better upgrading too.